src/lora_plan.erl

```diff
@@ -1206,8 +1210,8 @@ valid_uplink_freq(Plan, Freq) ->
             true;
         _ ->
             Ch = lora_region:f2uch(Region, Freq),
-            Freq2 = lora_region:uch2f(Region, Ch),
-            io:format("Freq=~w Ch=~w Freq2=~w ~n", [Freq, Ch, Freq2]),
+            _Freq2 = lora_region:uch2f(Region, Ch),
```

Comment:
Since the corresponding `io:format()` call is commented-out, no need for populating this value:

```suggestion
            %% Freq2 = lora_region:uch2f(Region, Ch),
```